### PR TITLE
Update UN/LOCODE from 2022-1 to 2022-2

### PR DESCRIPTION
**Pull request by vvarg229**

### CODE REVIEW

**Review by cthulhu-rider — COMMENTED** [review]
manual try succeeded!

lets bump minor version

**Comment by cthulhu-rider** [comment]
updated UN database may contain new records, fixes and removals, so it looks like a feature to me. This how it was versioned until now and I think it's pretty good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 Changelog for NeoFS LOCODE database
 
+## 0.3.1 - 2023-04-11
```
Comment (cthulhu-rider) [inline CHANGELOG.md:4]:
```suggestion
## 0.4.0 - 2023-04-11
```

**Review by cthulhu-rider — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vvarg229) [reply CHANGELOG.md]: Done

**Review by cthulhu-rider — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**